client/package.json

```
@@ -113,7 +113,7 @@
     "xml-beautifier": "^0.5.0"
   },
   "scripts": {
-    "develop": "NODE_OPTIONS=--max-old-space-size=4096 NODE_ENV=development gulp && webpack-dev-server",
+    "develop": "export NODE_OPTIONS=--max_old_space_size=4096 NODE_ENV=development gulp && webpack-dev-server",
```

```suggestion
    "develop": "export NODE_OPTIONS=--max_old_space_size=4096 NODE_ENV=development gulp && webpack-dev-server",
```

that export might 'eat' the gulp command (at least it does for me.)
```
$ export A=B ls
$
```

---

good catch